app/models/appointment.rb

```diff
@@ -60,6 +60,27 @@ def self.eligible_for_reminders(days_overdue: 3)
       .merge(Patient.contactable)
   end
 
+  def self.reminder_start_hour
```

Comment:
**Non-blocking:**

I read your comment about moving this `Communication`, which sounds better.

What happens when we need to stagger the time intervals at which we send these messages for different kinds of notifications or communications to avoid spamming? (which say, the notification framework suggests)?

Ideally we shouldn't tie relatively generic model methods against specific env variables, and instead pass them in.

Reply:
I think we can easily update the behavior of the `Communication.next_messaging_time` to spice up its returns with some entropy. The big win I see is one accessible place to ask this question, which eliminates the need to read and pass a time window through several layers of application code as arguments.

As for _where_ this logic should exist, that'll likely evolve as our requirements evolve. For now, `Communication` seems fine to me.